### PR TITLE
Fix importing old save shiny Pokemon

### PR DESCRIPTION
**Pull request by RedSparr0w**
Attached save below for testing:
[pokeclicker_2020-03-03T23_28_14.819Z.txt](https://github.com/pokeclicker/pokeclicker/files/4284275/pokeclicker_2020-03-03T23_28_14.819Z.txt)

### CODE REVIEW

File: src/scripts/Save.ts [hunk 1]
```diff
@@ -184,20 +184,21 @@ class Save {
...
-        //     }
-        // }
-        // if (converted.length > 0) {
-        //     Notifier.notify("You have gained the following shinies: " + converted, GameConstants.NotificationOption.success)
-        // }
+    public static convertShinies(list: Array<any>) {
+        const converted = [];
+        list = list.filter(p => p.shiny);
+        for (const pokemon of list) {
+            const id = +pokemon.id;
```
Comment (Ishadijcks) [inline src/scripts/Save.ts]: `+`?

File: src/scripts/Save.ts [hunk 2]
```diff
@@ -184,20 +184,21 @@ class Save {
...
-        //         converted.push(pokemon['name']);
-        //     }
-        // }
-        // if (converted.length > 0) {
-        //     Notifier.notify("You have gained the following shinies: " + converted, GameConstants.NotificationOption.success)
-        // }
+    public static convertShinies(list: Array<any>) {
+        const converted = [];
+        list = list.filter(p => p.shiny);
+        for (const pokemon of list) {
```
Comment (Ishadijcks) [inline src/scripts/Save.ts]: Does this work with the any type?

File: src/scripts/Save.ts (hunk 1 above)
Comment (RedSparr0w) [reply src/scripts/Save.ts]: Just insuring its a number

File: src/scripts/Save.ts (hunk 2 above)
Comment (RedSparr0w) [reply src/scripts/Save.ts]: Sure does, otherwise typescript throws an error expecting that everything in the array is a string.

**Pull request merged**